.gitignore

```diff
@@ -20,3 +20,4 @@ dist
 npm-debug*
 /.tox
 .idea
+package-lock.json
```

Comment:
It shouldn't be necessary to ignore this, though there are occasional issues.

Reply:
Sorry about that, will edit to remove!